tests/unit/CertificateGenerationService.unitTest.ts

```diff
@@ -25,21 +28,39 @@ import {Configuration} from "../../src/utils/Configuration";
 import { ITestStation } from "../../src/models/ITestStations";
 import { IDefectParent } from "../../src/models/IDefectParent";
 import { HTTPError } from "../../src/models/HTTPError";
+import queueEventPRS from "../resources/queue-event-prs.json";
+import queueEventPass from "../resources/queue-event-pass.json";
+import queueEventFail from "../resources/queue-event-fail.json";
 import Axios from "axios";
 
+jest.mock("@dvsa/cvs-microservice-common/feature-flags/profiles/vtx", () => ({
+  getProfile: mockGetProfile
+}));
+
 describe("Certificate Generation Service", () => {
   const sandbox = sinon.createSandbox();
   afterEach(() => {
     sandbox.restore();
   });
 
+  beforeAll(() => {
```

Comment:
Is this needed in a before each here? I think this is asking for issues later down the line, doing this for every test here